flang/include/flang/Lower/Runtime.h

```diff
@@ -202,7 +202,8 @@ class StaticMultimapView {
 enum class RuntimeEntryCode {
```

Comment:
Should this special enum be retired in favor of a table like IO and CharRT have?

More specifically, why have an extra enum value when we can use the call itself as a key into a runtime table? (This might mean having something like a MiscRT module, I suppose.)

Reply:
Good point, this enum is not bringing much, I got rid of it.